### DISCUSSION TITLE
Implementation of API

### DISCUSSION THREAD

**Discussion by gustaavik** [GriefPrevention/GriefPrevention#625] [Q&A] [answered]
**AS** a player/administrator/etc.
**I WANT** a feature
**BECAUSE** I want to be satisfied with my Minecrafting experience

I WANT an api, so I can use some of the features primarily the claiming system. 

I know how hard it is to create a claiming system with built in mysql.

**Comment by Jikoo** [comment]
What issues are you having using GriefPrevention's existing API? What feature are you asking to see in the API?

**Comment by gustaavik** [comment]
Can you link me to the api? or is it just by downloading the plugin, in that case I am not getting the options I was looking for.

**Comment by Jikoo** [comment]
> Can you link me to the api? or is it just by downloading the plugin, in that case I am not getting the options I was looking for.

Use the plugin as a dependency. GP does not offer a separate API artifact, but you can freely use exposed methods.

**Comment by RoboMWM** [comment, marked as answer]
If you're asking for docs, all we've got are a lot of code comments for now. v20 will change this.

Otherwise, what jikoo said.
